source/features/clean-repo-filelist-actions.tsx

```diff
@@ -38,12 +38,13 @@ function init(): void {
 			}
 
 			// This dropdown doesn't appear on `isSingleFile`
-			const downloadIcon = select('get-repo .octicon-download');
-			if (downloadIcon) {
+			// Remove `.octicon-download` in November
+			const codeIcon = select('get-repo :is(.octicon-code, .octicon-download)');
+			if (codeIcon) {
 				// Remove "Code" text next to it
-				downloadIcon.nextSibling!.remove();
+				codeIcon.nextSibling!.remove();
 
-				addTooltipToSummary(downloadIcon, 'Clone, open or download');
+				addTooltipToSummary(codeIcon, 'Clone, open or download');
```

Comment:
```js
addTooltipToSummary(codeIcon, codeIcon.nextSibling!.textContent);

// Remove "Code" text next to it
codeIcon.nextSibling!.remove();
```

Reply:
👎 it's better as is. "Code" isn't very useful as a tooltip

Reply:
I was about to reply roughly the same thing. Also doesn't make sense for me to use `textContent` instead of a literal `"Code”`.